pkg/reconcilers/deploymentconfig.go

```diff
@@ -283,3 +285,40 @@ func DeploymentConfigPodTemplateAnnotationsMutator(desired, existing *appsv1.Dep
 
 	return updated, nil
 }
+
+// DeploymentConfigArgsMutator ensures Args are reconciled
+func DeploymentConfigArgsMutator(desired, existing *appsv1.DeploymentConfig) (bool, error) {
+	updated := false
+
+	for i, desiredContainer := range desired.Spec.Template.Spec.Containers {
+		existingContainer := &existing.Spec.Template.Spec.Containers[i]
+
+		if !reflect.DeepEqual(existingContainer.Args, desiredContainer.Args) {
+			existingContainer.Args = desiredContainer.Args
+			updated = true
+		}
+	}
+
+	return updated, nil
+}
+
+// DeploymentConfigProbesMutator ensures probes are reconciled
+func DeploymentConfigProbesMutator(desired, existing *appsv1.DeploymentConfig) (bool, error) {
+	updated := false
+
+	for i, desiredContainer := range desired.Spec.Template.Spec.Containers {
+		existingContainer := &existing.Spec.Template.Spec.Containers[i]
+
+		if !reflect.DeepEqual(existingContainer.LivenessProbe, desiredContainer.LivenessProbe) {
+			existingContainer.LivenessProbe = desiredContainer.LivenessProbe
+			updated = true
+		}
+
+		if !reflect.DeepEqual(existingContainer.ReadinessProbe, desiredContainer.ReadinessProbe) {
+			existingContainer.ReadinessProbe = desiredContainer.ReadinessProbe
+			updated = true
+		}
+	}
+
+	return updated, nil
+}
```

Comment:
Could you add unit tests to `pkg/reconcilers/deploymentconfig_test.go` for the new functions, `DeploymentConfigArgsMutator` and `DeploymentConfigProbesMutator`?